tests/Unit/AddContactTest.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+use Inertia\Testing\AssertableInertia as Assert;
+
+class AddContactTest extends \Tests\TestCase
+{
+    use \Illuminate\Foundation\Testing\DatabaseMigrations;
+
+    public function test_add_contact_with_first_and_last_name()
+    {
+        $this->followRedirects = true;
+
+        $data = ['first_name' => 'Bob', 'last_name' => 'Smith'];
+
+        $this->assertCount(0, \App\Models\Contact::query()->where($data)->get());
+
+        $response = $this->postJson('/contact', $data);
+
+        $response->assertInertia(fn(Assert $page) => $page->component('Contact/Index'));
+
+        $this->assertGreaterThan(0, \App\Models\Contact::query()->where($data)->count());
+
+        $this->assertSame(
+            $data,
+            \App\Models\Contact::query()
```

Comment:
You can also put this response into a $sut variable (Subject under test)
then self::assertSame($data, $sut)